core/templates/dev/head/components/summary_tile/story_summary_tile_directive.html

```diff
@@ -0,0 +1,44 @@
+<a ng-href="<[getStoryLink()]>">
+  <div class="oppia-story-image">
+    <img src="https://image.ibb.co/iUeEUJ/Screenshot_from_2018_07_21_11_07_04.png" alt="">
+  </div>
+  <div class="oppia-story-details">
+    <div class="oppia-story-title">
+      <[getStoryTitle()]>
```

Comment:
For this and the description, possibly useful: https://toddmotto.com/angular-one-time-binding-syntax/

Reply:
Done.